inc/update.php

```diff
@@ -520,12 +520,17 @@ function largo_update_custom_less_variables() {
 
 /**
  * Checks for use of deprecated widgets and posts an alert
+ *
+ * Widgets that were checked in the past, but have been removed from the check after their files were removed form the theme:
+ * - 'largo-footer-featured'
+ * - 'largo-sidebar-featured'
+ * - 'largo-featured'
+ * @since 0.4
  */
 function largo_check_deprecated_widgets() {
 
 	$deprecated = array(
-		'largo-footer-featured' => 'largo_deprecated_footer_widget',
-		'largo-sidebar-featured' => 'largo_deprecated_sidebar_widget'
+		//'largo-footer-featured' => 'largo_deprecated_footer_widget',
```

Comment:
It's commented out as documentation, in the future case where we deprecate another widget but migrate its settings.